state-chain/cf-integration-tests/src/solana.rs

```diff
@@ -740,7 +740,7 @@ fn solana_ccm_execution_error_can_trigger_fallback() {
 
 			// Submit vote to witness: transaction success, but execution failure
 			let vote: SolanaElectionVote = BTreeMap::from_iter([(election_id,
-				AuthorityVote::Vote(CompositeVote::EE(TransactionSuccessDetails {
+				AuthorityVote::Vote(CompositeVote::D(TransactionSuccessDetails {
```

Comment:
note: I've made sure to run this migration *after* `remove_fee_tracking_migration.rs`